Admin-app/src/components/SignUp/index.js

```diff
@@ -314,6 +315,21 @@ export default function SignUp(props) {
               </Typography>
             </Box>
             </Modal>
+            <Modal
```

Comment:
@tusharraval102 can you create component for here too, or same component that will be created for admin signin!